app/routes/cdo/create/select-owner.js

```diff
@@ -0,0 +1,65 @@
+const { routes, views } = require('../../../constants/cdo/owner')
+const { routes: dogRoutes } = require('../../../constants/cdo/dog')
+const { getOwnerDetails, setOwnerDetails, setAddress } = require('../../../session/cdo/owner')
+const ViewModel = require('../../../models/cdo/create/select-owner')
+const { admin } = require('../../../auth/permissions')
+const { getPersons } = require('../../../api/ddi-index-api/persons')
+const { setInSession, getFromSession } = require('../../../session/session-wrapper')
+const Joi = require('joi')
+
+module.exports = [{
+  method: 'GET',
+  path: routes.selectOwner.get,
+  options: {
+    auth: { scope: [admin] },
+    handler: async (request, h) => {
+      const ownerDetails = getOwnerDetails(request)
+      const ownerResults = await getPersons(ownerDetails)
+
+      setInSession(request, 'persons', ownerResults)
```

Comment:
Just me musing out-loud here ... would it be better to NOT store the results in the session but use the person Reference as the radio group value. Then you use the radio group value (personReference) in looking up the person here?
Due to needing to get this out, I suggest not changing anything.

Reply:
I did consider that, the pattern was to use indexes so I decided to stay with that